components/App.vue

```diff
@@ -168,19 +60,43 @@ if (process.client) pageWidth.value = window.innerWidth;
 
 const isUsingDayMaxEventRows = useState('isUsingDayMaxEventRows', () => true);
 
-const updateWeekNumbers = () => { 
-  return getWindowWidth() < 350 ? false : true 
+const updateWeekNumbers = () => {
+  return getWindowWidth() < 350 ? false : true
 };
 // -1 indicates that there is no limit.
 const updateDayMaxEventRows = () => { return isUsingDayMaxEventRows.value ? -1 : Math.floor(getWindowHeight() / 75) };
 
+const calendarOptions = ref<CalendarOptions | undefined>()
+
+const disabledEventSources = new Map<string, EventSourceInput>()
+
+function enableEventSource(name: string) {
+  if (!calendarOptions.value?.eventSources) return
+  if (calendarOptions.value.eventSources.some(eventSource => name === eventSource.name)) return
+  const source = disabledEventSources.get(name)
+  if (source) calendarOptions.value.eventSources.push(source)
+}
+
+function disableEventSource(name: string) {
+  if (!calendarOptions.value?.eventSources) return
+  const newEventSources: EventSourceInput[] = []
+
+  calendarOptions.value.eventSources.forEach(eventSource => {
+    if (name === eventSource.name) {
+      disabledEventSources.set(name, eventSource)
+    } else {
+      newEventSources.push(eventSource)
+    }
+  })
+
+  calendarOptions.value.eventSources = newEventSources
+}
+
```

Comment:
The core of this PR. Basically, adding/removing event sources from `calendarOptions.value.eventSources` adds/removes events from that source.